frontend/rolecall/src/app/cast/cast-editor-v2.component.ts

```diff
@@ -81,7 +81,7 @@ export class CastEditorV2 implements OnInit {
     let autoSelectFirst = false;
     this.selectedPiece = piece;
     this.updateFilteredCasts();
-    if (this.selectedPiece && this.selectedPiece.uuid !== piece.uuid) {
```

Comment:
Is this change required for the new pipe implementation? If not, could it be moved to a separate PR?

Reply:
It was a display bug I discovered while verifying that the pipe worked. It was an easy fix, so I though I could sneak it in.